toolchain/check/import_ref.cpp

```diff
@@ -1392,6 +1392,26 @@ static auto TryResolveTypedInst(ImportRefResolver& resolver,
   return ResolveResult::Done(resolver.local_constant_values().Get(inst_id));
 }
 
+static auto TryResolveTypedInst(ImportRefResolver& resolver, SemIR::Vtable inst,
+                                SemIR::InstId import_inst_id) -> ResolveResult {
+  auto type_const_id = GetLocalConstantId(resolver, inst.type_id);
+  auto virtual_functions =
+      GetLocalInstBlockContents(resolver, inst.virtual_functions_id);
+  if (resolver.HasNewWork()) {
+    return ResolveResult::Retry();
+  }
+
+  auto virtual_functions_id = GetLocalCanonicalInstBlockId(
+      resolver, inst.virtual_functions_id, virtual_functions);
+  auto inst_id = resolver.local_context().AddInstInNoBlock(
+      resolver.local_context().MakeImportedLocAndInst<SemIR::Vtable>(
+          AddImportIRInst(resolver, import_inst_id),
+          {.type_id =
+               resolver.local_context().GetTypeIdForTypeConstant(type_const_id),
+           .virtual_functions_id = virtual_functions_id}));
+  return ResolveResult::Done(resolver.local_constant_values().Get(inst_id));
```

Comment:
`inst_id` comes from `AddInstInNoBlock` so looking it up in `local_constant_values()` looks wrong to me. Should it be `local_insts()`?